rules-reviewed/openliberty/liberty_LibertyList.windup.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="liberty_LibertyList">
+    <metadata>
+        <dependencies>
```

Comment:
It would be great to some a pair of lines of description for the ruleset.
```suggestion
        <description>
            Provide a description
        </description>
        <dependencies>
```